src/main/java/com/dezorganizacja/bridge/view/graphical/Controller.java

```diff
@@ -0,0 +1,11 @@
+package com.dezorganizacja.bridge.view.graphical;
+
+import com.dezorganizacja.bridge.presenter.MainPresenter;
+
+public class Controller {
+    protected MainPresenter presenter;
```

Comment:
Jako uzupełnienie dodam jeszcze, że funkcja `setState` na `MainPresenterze` docelowo nie ma zbyt wiele sensu. Zamiast wołać metody o abstrakcyjnych nazwach dostaniesz po prostu akcje w stylu: "newGame", "exit" które z założenia zmienią stan i które z założenia będą miały przewidywalny efekt dla Twoich kontrolerów. A skoro tak, to kontrolery w **większości przypadków** będą potrafiły od razu zareagować, bez czekania na jakiegoś obserwera czy coś.

`setState` ma sens tylko i wyłącznie na poziomie tego PR, gdzie dopiero zaczynamy robić jakieś interakcje z interfejsem i nie mamy całej reszty. To jest super i w zasadzie lepiej niż myślałem, że zrobisz. :dancer: